Source/Turbo/Navigator/Extensions/VisitProposalExtension.swift

```diff
@@ -26,6 +26,14 @@ public extension VisitProposal {
         properties["pull_to_refresh_enabled"] as? Bool ?? true
     }
 
+    var modalSheetDismissGestureEnabled: Bool {
+        if let dismissEnabled = properties["modal_sheet_dismiss_gesture_enabled"] as? Bool {
```

Comment:
Since this gesture applies to all modal styles (not just the `pageSheet` and `formSheet` styles), maybe it's be more clear and less verbose to drop the `_sheet_` part of the name and go with:
```
modal_dismiss_gesture_enabled
```

What do you think?